mhs/inbound/main.py

```diff
@@ -85,9 +85,10 @@ def start_inbound_server(local_certs_file: str, ca_certs_file: str, key_file: st
     healthcheck_application = tornado.web.Application([
         ("/healthcheck", healthcheck_handler.HealthcheckHandler)
     ])
-    healthcheck_application.listen(80)
+    server_port = int(config.get_config('INBOUND_SERVER_PORT', default='80'))
+    healthcheck_application.listen(server_port)
 
-    logger.info('011', 'Starting inbound server')
+    logger.info('011', 'Starting inbound server at port ' + str(server_port))
```

Comment:
I think our logger is setup to accept template strings, for example `logger.info('0010', 'Returning response with {HttpStatus}', {'HttpStatus': status})`. You could also use an f-string here `f'Starting inbound server at port {server_port}'`. Using + for string concatenation in Python can be unpredictable.

Reply:
good point